src/main/java/net/dv8tion/jda/api/entities/IMemberContainer.java

```diff
@@ -19,7 +19,13 @@
 import javax.annotation.Nonnull;
 import java.util.List;
 
-//TODO-v5: Need Docs
+/**
+ * Represents a {@link GuildChannel} that is capable of containing members.
+ *
+ * Implementations interpret this meaning as best applies to them.
```

Comment:
```suggestion
 * Implementations interpret this meaning as best applies to them.
 * For example, {@link TextChannel TextChannels} implement this as the {@link Member members} that have {@link Permission.VIEW_CHANNEL} in the channel, whereas {@link VoiceChannel VoiceChannels} implement this to define what {@link Member members} are currently connected to the channel.
```
Please double check that this `{@link}` references work.